docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/installer.md

```diff
@@ -0,0 +1,398 @@
+Services:
+  Systemd service |       Version |  Port |                            Log File Locations |  Running Status |
+         postgres |         10.23 |  5432 |          /opt/yugabyte/data/logs/postgres.log |         Running |
+       prometheus |        2.42.0 |  9090 |  /opt/yugabyte/data/prometheus/prometheus.log |         Running |
+      yb-platform |  2.19.0.0-b59 |   443 |       /opt/yugabyte/data/logs/application.log |         Running |
+```
+
+### Upgrade
+
+To upgrade using YBA Installer, first download the version of YBA Installer corresponding to the version of YugabyteDB Anywhere you want to upgrade to. See [Download YBA Installer](#download-yba-installer). Upgrade works similarly to the install workflow, by first running preflight checks to validate the system is in a good state. When ready to upgrade, run the upgrade command from the untarred directory of the target version of the YBA upgrade:
+
+```sh
+$ sudo ./yba-ctl upgrade
+```
+
+The upgrade takes a few minutes to complete. When finished, use the [status command](#service-management) to verify that YugabyteDB Anywhere has been upgraded to the target version.
+
+### Backup and restore
+
+YBA Installer also provides utilities to take full backups of the YugabyteDB Anywhere state (not YugabyteDB however) and later restore from them. This not only includes data seen in YBA for your universes, but also metrics stored in Prometheus.
+
+To perform a backup, provide the full path to the directory where the backup will be generated. The `createBackup` command creates a timestamped `tgz` file for the backup. For example:
+
+```sh
+$ sudo yba-ctl createBackup ~/test_backup
+$ ls test_backup/
+```
+
+```output
+backup_23-04-25-16-54.tgz  version_metadata_backup.json
+```
+
+To restore from the same backup, use the `restoreBackup` commmand:
+
+```sh
+$ sudo yba-ctl restoreBackup ~/test_backup/backup_23-04-25-16-64.tgz
+```
+
+### Clean (uninstall)
+
+To uninstall a YBA instance, YBA Installer also provides a clean functionality with two modes. By default, `clean` removes the YugabyteDB Anywhere software, but keeps any data such as PostgreSQL or Prometheus information. You can also run `clean` with the `–all` flag to delete all data.
+
+```sh
+$ sudo yba-ctl clean
+```
+
+```output
+INFO[2023-04-24T23:58:13Z] Uninstalling yb-platform
+INFO[2023-04-24T23:58:14Z] Uninstalling prometheus
+INFO[2023-04-24T23:58:14Z] Uninstalling postgres
+```
+
+```sh
+$ sudo yba-ctl clean -all
+```
+
+```output
+--all was specified. This will delete all data with no way to recover. Continue? [yes/NO]: y
+INFO[2023-04-24T23:58:13Z] Uninstalling yb-platform
+INFO[2023-04-24T23:58:14Z] Uninstalling prometheus
+INFO[2023-04-24T23:58:14Z] Uninstalling postgres
+```
+
+## Migrate a YugabyteDB Anywhere installation
+
+You can migrate a YugabyteDB Anywhere installation from a Replicated-based deployment to a YBA Installer deployment.
+
+### Migrate from Replicated to YBA Installer
+
+## Running yba-ctl commands
```

Comment:
Provide a link to this section right after https://deploy-preview-17176--infallible-bardeen-164bc9.netlify.app/preview/yugabyte-platform/install-yugabyte-platform/install-software/installer/#download-yba-installer .